mirrord/layer/src/file/hooks.rs

```diff
@@ -804,6 +899,7 @@ pub(crate) unsafe fn enable_file_hooks(hook_manager: &mut HookManager) {
     );
 
     replace!(hook_manager, "openat", openat_detour, FnOpenat, FN_OPENAT);
+    replace!(hook_manager, "openat64", openat_detour, FnOpenat, FN_OPENAT);
```

Comment:
It'd be more preferable to have different detours, for case where `openat` resolves but `openat64` doesn't (as it assigns it to the `FN_OPENAT` (we had cases like this, that's why inode hooks are in cfg, which tbh we should just have similar logic)

Reply:
Changed.